src/backend/main.ts

```diff
@@ -513,6 +522,17 @@ ipcMain.handle('checkDiskSpace', async (event, folder) => {
   })
 })
 
+ipcMain.handle(
+  'isFrameless',
+  () =>
```

Comment:
maybe, instead of exporting `getWindowProps` from mainWindow, we could export an `isFrameless` function that would to this logic, I think it will look cleaner and we don't export the window props variable

Reply:
Well in that case, I would almost prefer to augment the `BrowserWindow` class in our main window with a method `isFrameless()` so I can write `getMainWindow()?.isFrameless()` and do without any additional exports, but I'm not sure if it's worth all of the TypeScript incantations required to make that happen.